main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
```

Comment:
Just a suggestion to initialize the loop variable to start at 0 instead of 1 for readability purposes. The way it's written `loop = 1` signifies that the loop has already repeated once

Reply:
```suggestion
loop = 0
```